lib/jnpr/junos/device.py

```diff
@@ -833,11 +833,11 @@ def __init__(self, *vargs, **kvargs):
             self._conf_ssh_private_key_file = None
             # user can get updated by ssh_config
             self._ssh_config = kvargs.get('ssh_config')
+            self._sshconf_lkup()
             # but if user or private key is explicit from call, then use it.
             self._auth_user = kvargs.get('user') or self._conf_auth_user or \
                 self._auth_user
-            self._ssh_private_key_file = kvargs.get('ssh_private_key_file') \
-                or self._conf_ssh_private_key_file
+            self._ssh_private_key_file = kvargs.get('ssh_private_key_file')
```

Comment:
why "or self._conf_ssh_private_key_file" was removed. If user has configured private key file in ssh config file, by removing the code the value will be set to None only.